modules/ROOT/pages/linking.adoc

```diff
@@ -135,6 +134,61 @@ ordersByOrderId (orderId: “123”) {
 }
 --
 
+== Creating a Link Between Object Types Using Composite Keys
+
+The default query method for a type can also use _composite keys_.
+
+Consider a target type `Customer`, which uses composite keys for its default query method: `customerId` and `customerStatus`.
+
+[source]
+--
+customerDetail (id: string!, status): Customer
```

Comment:
the field `status` must be mandatory and must have a type like: `customerDetail (id: string!, status: string!): Customer`